File: app/src/main/java/com/dewonderstruck/apps/ashx0/ui/blog/detail/BlogDetailActivity.java
package com.dewonderstruck.apps.ashx0.ui.blog.detail;

import android.os.Bundle;


import com.dewonderstruck.apps.ashx0.R;

import com.dewonderstruck.apps.ashx0.ui.common.PSAppCompactActivity;
import com.dewonderstruck.apps.ashx0.databinding.ActivityBlogDetailBinding;

import androidx.databinding.DataBindingUtil;

public class BlogDetailActivity extends PSAppCompactActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        ActivityBlogDetailBinding binding = DataBindingUtil.setContentView(this, R.layout.activity_blog_detail);

        initUI(binding);

    }

    private void initUI(ActivityBlogDetailBinding binding) {

        // Toolbar
        initToolbar(binding.toolbar, getResources().getString(R.string.blog_detail__title));

        // setup Fragment
        setupFragment(new BlogDetailFragment());

    }
}

File: app/src/main/java/com/dewonderstruck/apps/ashx0/ui/blog/detail/BlogDetailActivity.kt
package com.dewonderstruck.apps.ashx0.ui.blog.detail

import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.dewonderstruck.apps.ashx0.R
import com.dewonderstruck.apps.ashx0.databinding.ActivityBlogDetailBinding
import com.dewonderstruck.apps.ashx0.ui.common.PSAppCompactActivity

class BlogDetailActivity : PSAppCompactActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding: ActivityBlogDetailBinding = DataBindingUtil.setContentView(this, R.layout.activity_blog_detail)
        initUI(binding)
    }

    private fun initUI(binding: ActivityBlogDetailBinding) {

        // Toolbar
        initToolbar(binding.toolbar, resources.getString(R.string.blog_detail__title))

        // setup Fragment
        setupFragment(BlogDetailFragment())
    }
}